### DISCUSSION TITLE
[BUG] Get-PnPTenant only works with DeviceLogin in PS7

### DISCUSSION THREAD

**Discussion by paylord** [pnp/powershell#3465] [General]
### Reporting an Issue or Missing Feature
Get-PnPTenant in PS7 fails when connected using Interactive mode

### Expected behavior 
Expect Get-PnPTenant to return org properties

### Actual behavior 
In PS7.3
Connect-PnPOnline -Url $adminUrl -Interactive # Seems to work fine, can run Get-PnPTenantSite etc
Get-PnPTenant
Get-PnPTenant: Attempted to perform an unauthorized operation.

The above works fine in PS5.

Works fine in PS7 if I use DeviceLogin:
Connect-PnPOnline -Url $adminUrl -DeviceLogin
Get-PnPTenant # returns properties as expected

### Steps to reproduce behavior
See above

### What is the version of the Cmdlet module you are running?
PS7 = 2.2.0
PS5 = 1.12.0

### Which operating system/environment are you running PnP PowerShell on?
- [ X ] Windows

**Comment by veronicageek** [comment]
@paylord - I have no problem using PS 7.3 and PnP PowerShell 2.2.0
Either it's machine config issue, permission issue, or this happened at a point in time. Please try again and feedback to us.

![image](https://github.com/pnp/powershell/assets/25181757/ea3a1174-58a9-4f02-ab92-346ba1fb4cd1)

**Comment by paylord** [comment]
@veronicageek - thanks for your response.

I tested again this morning and get the same issue.

However, I also tested against a different tenant and -Interactive mode works fine.  This suggests it may be a specific tenant setting.

I also now have a PFX certificate for the failing tenant so tested that and again it works fine.

So at the moment, it looks like it is just an issue with -Interactive mode on a specific tenant.

Do you have any suggestions what may be causing this?

**Comment by gautamdsheth** [comment]
Since this issue is external to PnP PowerShell, am moving this to discussion.

The difference could be in the policies applied in that tenant or issue with the Azure AD app configuration or something else , can't say for sure
